src/patterns/components/dictionary/striped.hbs

```diff
@@ -0,0 +1,34 @@
+---
+order: 2
+notes: This variation is intended for cases where the list of key-value pairs is especially long
```

Comment:
I don't know what you mean by "especially long". Like lots of entries in the list or the lines are long because it's in a wide container?

Reply:
If it's the former, I would write: "This variation is for cases where the number of items in the Dictionary exceeds [some number]. It provides a visual cue connecting the key to its value."

If the later, I would write: "This variation is for cases where the width of the Dictionary makes it difficult scan. It provides a visual cue connecting the key to its value."